satellite/satellitedb/migrate.go

```diff
@@ -1121,6 +1121,13 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					`DROP TABLE certRecords CASCADE`,
 				},
 			},
+			{
+				Description: "Add piece_count column to nodes table",
+				Version:     53,
+				Action: migrate.SQL{
+					`ALTER TABLE nodes ADD piece_count INTEGER NOT NULL DEFAULT 0;`,
```

Comment:
There isn't a default specified in the DBX model, so we have to specify it there or remove the `DEFAULT` constraint in a new SQL migration after this one.

Reply:
I looked through the dbx documentation and I don't see anything about setting defaults. Are you sure that's a thing?

Reply:
Nevermind, @mobyvb pointed out `autoinsert`.